src/ci/docker/host-x86_64/mingw-check/validate-error-codes.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Checks that no error code explanation is removed.
+
+set -euo pipefail
+
+if (git log HEAD..."$BASE_COMMIT" --diff-filter=D --summary | grep delete \
```

Comment:
Shouldn't the order be inverted?
```suggestion
if (git log "$BASE_COMMIT"...HEAD --diff-filter=D --summary | grep delete \
```